### PR TITLE
[name change] Use LsbLargerThanMsbException

### PR DESCRIPTION
**Pull request by seldridge**
- Rename `LsbLargerThanMsb` to `LsbLargerThanMsbException` (@kevintownsend)

### CODE REVIEW

File: src/main/scala/firrtl/passes/Checks.scala [hunk 1]
```diff
@@ -51,7 +51,7 @@ object CheckHighForm extends Pass {
     s"$info: A single module must be named $name.")
   class NegArgException(info: Info, mname: String, op: String, value: Int) extends PassException(
     s"$info: [module $mname] Primop $op argument $value < 0.")
-  class LsbLargerThanMsb(info: Info, mname: String, op: String, lsb: Int, msb: Int) extends PassException(
+  class LsbLargerThanMsbException(info: Info, mname: String, op: String, lsb: Int, msb: Int) extends PassException(
```
Comment (kevintownsend) [inline src/main/scala/firrtl/passes/Checks.scala]: Nit: If the line limit is 100 characters you are over. Line break just before the "extends" keyword.

File: src/main/scala/firrtl/passes/Checks.scala [hunk 2]
```diff
@@ -90,7 +90,8 @@ object CheckHighForm extends Pass {
           correctNum(Option(1), 2)
           val (msb, lsb) = (e.consts(0).toInt, e.consts(1).toInt)
           if (lsb > msb) {
-            errors.append(new LsbLargerThanMsb(info, mname, e.op.toString, lsb, msb))
+            errors.append(
```
Comment (kevintownsend) [inline src/main/scala/firrtl/passes/Checks.scala]: Nit: If the line limit is 100 characters you don't need a line break.

File: src/main/scala/firrtl/passes/Checks.scala (hunk 1 above)
Comment (jackkoenig) [reply src/main/scala/firrtl/passes/Checks.scala]: I think 120 is the normal Scala-style limit, I've been moving toward 100 but the official limit remains 120

File: src/main/scala/firrtl/passes/Checks.scala (hunk 2 above)
Comment (seldridge) [reply src/main/scala/firrtl/passes/Checks.scala]: I'm in agreement. This is my internal >80 alarm going off. I need to setup per-project settings...

**Pull request merged**